arrows/ocv/resection_camera.cxx

```diff
@@ -58,9 +62,10 @@ ::get_configuration() const
     vital::algo::resection_camera::get_configuration();
   d_->get_configuration( config );
   config->set_value( "reproj_accuracy", d_->reproj_accuracy,
-                     "desired re-projection positive accuracy" );
+                     "desired re-projection positive accuracy for inlier points" );
   config->set_value( "max_iterations", d_->max_iterations,
                      "maximum number of iterations to run optimization [1, INT_MAX]" );
+  // TODO: focal_scales
```

Comment:
Adding options to set the actual focal scales would be nice, but at a minimum, I think what this branch needs for now is a way to disable the use of focal scales through configuration.  It is often the case that you know the intrinsic model and want the best resectioned camera using that exact intrinsic model, without allowing the focal length to change.  So we need a way of turning off the multiple focal scales.  If we can just set the focal scales in the config then we could set this with just one scale of 1.0.  If that's too trick for now, we should at least have a boolean option to enable or disable these scales.

Reply:
When discussed with BC and ML this changed seemed like a good idea for our BA needs, which currently relies just on trusted tracks, but still uses the notion of fixed landmarks which are supplied via an index. Given no other needs, I suppose this change can be dropped.